app/controllers/flood_risk_engine/enrollments/addresses_controller.rb

```diff
@@ -0,0 +1,107 @@
+require_dependency "flood_risk_engine/application_controller"
+
+module FloodRiskEngine
+  module Enrollments
+    class AddressesController < ApplicationController
```

Comment:
Why do we need a whole Controller just for  the manual address page ? especially given the insistence that the address service lookup stuff stick to the established StepController form/show/update cycle